### PR TITLE
Add support for wmbus water meter Maddalena

### PR DESCRIPTION
**Pull request by jedi7**
* Add actual water measure
* Add measure of last months

Buf cols must be increased because the wmbus payload is quite long (over 2048 bits)

There are still some unknown data, which will need more reverse engineering.

### CODE REVIEW

**Comment by jedi7** [comment]
Recorded data:
[wbus_maddalena.zip](https://github.com/merbanan/rtl_433/files/5796012/wbus_maddalena.zip)

**Comment by merbanan** [comment]
Looks ok, can you fix the remaining style checks?

And can you also contribute this to our competition? https://github.com/weetmuts/wmbusmeters

**Comment by jedi7** [comment]
I can :) but I do not know which, can you point me?
I will look at the wmbusmeters, maybe they are more far in the reverse engineering of the meters.

**Comment by merbanan** [comment]
Style issues are fixed now. I just need to look through the increase of the columns. And yeah the wmbusmeters project has reverse-engineered lots of meters. They want to support your meter and they want the datagram.

File: include/bitbuffer.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 #include <stdint.h>
 
-#define BITBUF_COLS 256 // Number of bytes in a column
+#define BITBUF_COLS 512 // Number of bytes in a column
```
Comment (zuckschwerdt) [inline include/bitbuffer.h:17]: Can we make that 400 columns (if I check correctly you need 392) and add a comment that "wmbus needs at least 392"

Comment (jedi7) [reply include/bitbuffer.h:17]: do we need so low limits? It is not running on embedded system, so 256B of memory is ok I hope :)

Comment (zuckschwerdt) [reply include/bitbuffer.h:17]: actually +256B times BITBUF_ROWS, and we do aim to run on low-end embedded (e.g. old routers) ;)

Comment (jedi7) [reply include/bitbuffer.h:17]: IMHO then we need at least ~437B  (( (256+16*2+3)*12)/8 ), when the data len will be 256 + 16*2 CRC + 3B

**Comment by jedi7** [comment]
guys, I added also commit which fixes the length of the value. Is possible to test this change also on another wmbus, because the change can hit also the already present temperature meters.

**Comment by jedi7** [comment]
this PR can be replaced by "better" PR https://github.com/merbanan/rtl_433/pull/1610

**Pull request closed without merging**